### DISCUSSION TITLE
How to use FYNE to implement RTSP or RTMP live streaming

### DISCUSSION THREAD

**Discussion by mynameiscuining** [fyne-io/fyne#5273] [Q&A]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same problem, before opening a new one.
- [X] This issue only relates to a single bug. I will open new issues for any other problems.

### Describe the bug

How to use FYNE to implement RTSP or RTMP live streaming

### How to reproduce

How to use FYNE to implement RTSP or RTMP live streaming

### Example code

How to use FYNE to implement RTSP or RTMP live streaming

### Fyne version

2.5

### Go compiler version

1.23.2

### Operating system and version

win11

**Comment by mynameiscuining** [comment]
demo link？？？？

**Comment by andydotxyz** [comment]
Please use discussions or one of our support channels for asking questions, issues are for reporting bugs.

I would imagine the main challenge for a media stream will be the decoding of the data, but I'm not sure what you have tried or how far you got in development.

**Reply by mynameiscuining** [reply]
how to decode h264 without ffmpeg ，It's a hassle to configure various environment variables
